src/Sulu/Bundle/AdminBundle/Resources/translations/sulu/backend.en.xlf

```diff
@@ -1434,6 +1434,22 @@
                 <source>labels.successfully-saved</source>
```

Comment:
should have the same id in english and german `a504f522de1456524142c0b4820e29a5f `